alleblock.txt

```diff
@@ -216,6 +216,7 @@
 	allegro.pl##div[data-box-name="allegro.locationSuggest"]
 	allegro.pl##div[data-box-name="premium.with.dfp"]
 	allegro.pl##div[data-box-name="cart_reco_bought"]
+	allegro.pl##[data-analytics-view-label="showSponsoredItems"]
```

Comment:
W sumie to tak dopiero zwróciłem uwagę na jeszcze jeden szczegół - on zawsze wstawia `##div` na początku filtrów, nie wiem po co, no ale wstawia, oczywiście to nie znaczy, że i my mamy go papugować i wstawiać, w sumie jak będzie chciał to se sam dostawi jeśli mu to do czegoś potrzebne, no chyba, że wolisz mu to wstawić, żeby zachować zgodność z jego sposobem tworzenia filtrów, zrobisz jak uważasz.